error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -144,6 +144,18 @@ Mono<S> after(Mono<T> mono, S object) {
     }
   }
 
+  static final class FluxTakeGenerationLimit<T> {
```

Comment:
```suggestion
  static final class FluxTake<T> {
```

Reply:
We use content of the after-template to determine how we should name the Refaster rule. We can simply write `FluxTake` here as we see in the after-template. 

(We are working on a naming algorithm that will be enforced.)